camayoc/tests/qcs/conftest.py

```diff
@@ -0,0 +1,20 @@
+"""Pytest customizations and fixtures for the quipucords tests."""
+import pytest
+
+from camayoc import api
+
+
+@pytest.fixture
```

Comment:
Default scope is at the function level, right? Would this affect your test execution if you were to run the suite in parallel?

Reply:
Yes, this would. To make only delete the entities created by the test, I think maybe it should just be a function and the test can pass as arguments list of credential id's and profile id's

Reply:
I ended up re-factoring this to `yield` a list of ids to cleanup to the test that uses the fixture, and to then delete them after the test ends.